src/main/java/com/minecolonies/commands/ISubCommand.java

```diff
@@ -0,0 +1,27 @@
+package com.minecolonies.commands;
+
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * Override this to specify a sub command.
+ */
+public interface ISubCommand
+{
+
+    @NotNull
+    String getCommandUsage(@NotNull final ICommandSender sender);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)